README.md

```diff
@@ -74,14 +77,19 @@ Pipeline execution
     - Install the `tkn` CLI tool
     - Execute the `tkn pipeline start` command with the required parameters
 
-Trigger nightly pipeline manually
+Useful commands
 ---
+* Trigger nightly pipeline manually
 ```shell
 kubectl create job --from=cronjob/trigger-nightly-pipeline trigger-nightly-pipeline-$(date +%d.%m)-$(whoami)-manual -n ${PIPELINE_NAMESPACE}
 ```
 
-Setup automatic cleanup of old PipelineRun's every week
----
+* Set default dns configuration for Tekton pods
+```shell
+kubectl patch configmap config-defaults --type merge -p '{"data": {"default-pod-template": "dnsConfig:\n  nameservers:\n    - 1.2.3.4\n    - 5.6.7.8\ndnsPolicy: None"}}' -n openshift-pipelines
```

Comment:
`config-defaults` is managed by some other resource called `TektonInstallerSet/pipeline-main-static-ccf6d` Is this the right place to patch the nameservers?

Reply:
yes it is https://tekton.dev/docs/pipelines/additional-configs/#customizing-basic-execution-parameters

Reply:
But we do not use upstream Tekton but Openshift Pipelines. I would maybe add it to `spec.pipeline.default-pod-template`

According to doc: https://docs.redhat.com/en/documentation/red_hat_openshift_pipelines/1.18/html-single/installing_and_configuring/index#op-optional-configuration-fields_customizing-configurations-in-the-tektonconfig-cr

Reply:
Looks like both work, and it gets merged there from `config-defaults` automatically. I'm not against using the `TektonConfig` only for our pipeline settings